File: CodingSamples/Java/Runtime/ReflectionTest1/Shop.java
record Item(String name, String brand) {}

record Customer(String id, double purchase, int rating) {}

class Shop {

	public static Item getCommonItem() {
		return new Item("ssd", "samsung");
	}

	public static Customer getBestCustomer() {
		return new Customer("ganesh", 58000, 5);
	}
}

File: CodingSamples/DotNet/Runtime/ReflectionTest1/DemoApp/Shop.cs

record Item(string Name, string Brand);

record Customer(string Id, decimal Purchase, int Rating, string City = "Mumbai");

class Shop
{
	public readonly static Item BestItem = new Item("ssd", "samsung");

	public readonly static Customer BestCustomer = new Customer("ganesh", 58000, 5);

}